src/main/plugins/websearch-plugin/websearch-plugin.ts

```diff
@@ -83,6 +84,20 @@ export class WebSearchPlugin implements ExecutionPlugin {
         return this.urlExecutor(searchResultItem.executionArgument);
     }
 
+    public autoComplete(searchResultItem: SearchResultItem): string {
+        const searchUrl = searchResultItem.executionArgument.match(/^([^:]+:\/\/[^\/]+)\//) ?
+            RegExp.$1 :
+            searchResultItem.executionArgument;
+
+        const foundWebSearchEngine = this.config.webSearchEngines.find((websearchEngine) => {
+            return websearchEngine.url.includes(searchUrl);
+        });
+
+        const prefix = foundWebSearchEngine ? foundWebSearchEngine.prefix : "";
+
+        return `${prefix}${searchResultItem.name} `;
```

Comment:
Is the whitespace after `${searchResultitem.name}` intended?

Reply:
To me it would make more sense to have the autocompletion result without it. 🤔

Reply:
It indeed is intended. The idea behind it is, that you can continue refining your search string without having to hit space again. If you wouldn't want to refine your search string, you wouldn't have used tab, but would have hit `enter` in the first place.

Additionally it is most useful for using duckduckgos bangs.

Reply:
Ok I see, makes sense.